### PR TITLE
[qr] Rename Sailfish to Amber. Contributes to JB#54724

### PR DESCRIPTION
**Pull request by HandsomeJaack**
Depends on https://github.com/sailfishos/sailjail-permissions/pull/89

### CODE REVIEW

**Review by okodron — APPROVED** [review]

File: service/dbus/amber.zxing.service [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [D-BUS Service]
-Name=org.sailfishos.zxing
+Name=amber.zxing
```
Comment (rainemak) [inline service/dbus/amber.zxing.service]:
```suggestion
Name=org.amberapi.zxing
```
Please use org.amberapi for the service name.

**Review by okodron — APPROVED** [review]

**Review by rainemak — APPROVED** [review]

File: plugin/plugin.cpp [hunk 2]
```diff
@@ -12,15 +12,15 @@
...
+    Q_PLUGIN_METADATA(IID "amber.qrfilter")
 
 public:
     QrFilterPlugin() = default;
     virtual ~QrFilterPlugin() = default;
 
     void registerTypes(const char *uri) override
     {
-        Q_ASSERT(QLatin1String(uri) == QStringLiteral("org.sailfishos.qrfilter"));
+        Q_ASSERT(QLatin1String(uri) == QStringLiteral("amber.qrfilter"));
```
Comment (pvuorela) [inline plugin/plugin.cpp]: Wait a second, shouldn't this be Amber.QrFilter? Or should have been already uppercase sailfish for what i can tell.

Comment (rainemak) [reply plugin/plugin.cpp]: Good catch

Comment (HandsomeJaack) [reply plugin/plugin.cpp:23]: So, `Amber.QrFilter` should be instead?

Comment (HandsomeJaack) [reply plugin/plugin.cpp]: Fixed, thx

Comment (okodron) [reply plugin/plugin.cpp]: Pekka, thanks

**Review by okodron — APPROVED** [review]

**Pull request merged**